app/views/cloud_volume/attach.html.haml

```diff
@@ -1,61 +1,4 @@
-               "miq-form"      => true,
-               "model"         => "vm.cloudVolumeModel",
-               "model-copy"    => "vm.modelCopy"}
-  = render :partial => "layouts/flash_msg"
-  %h3
-    = _('Attach Volume')
-  .form-horizontal
-    .form-group{"ng-class" => "{'has-error': angularForm.vm_id.$invalid}"}
-      %label.col-md-2.control-label{"for" => "vm_id"}
-        = _('Instance')
-      .col-md-8
-        = select_tag("vm_id",
-                      options_for_select([["<#{_('Choose')}>", nil]] + @vm_choices.sort, disabled: ["<#{_('Choose')}>", nil]),
-                      "ng-model"                    => "vm.cloudVolumeModel.vm_id",
-                      "required"                    => true,
-                      :checkchange                  => true,
-                      "selectpicker-for-select-tag" => "")
-    .form-group{"ng-class" => "{'has-error': angularForm.device_path.$invalid}"}
-      %label.col-md-2.control-label
-        - if @volume.ext_management_system.type == 'ManageIQ::Providers::Amazon::StorageManager::Ebs'
-          = _('Device Mountpoint')
-        - else
-          = _('Device Mountpoint (optional)')
-      .col-md-8
-        %input.form-control{:type          => "text",
-                            :name          => "device_path",
-                            'ng-model'     => "vm.cloudVolumeModel.device_path",
-                            'ng-maxlength' => 128,
-                            "ng-required"  => "vm.cloudVolumeModel.emstype == 'ManageIQ::Providers::Amazon::StorageManager::Ebs'",
-                            :checkchange   => true}
-        %span.help-block{"ng-show" => "angularForm.device_path.$error.required"}
-          = _("Required")
-
-  .clearfix
-  .pull-right.button-group.edit_buttons
-    %miq-button{:name      => t = _("Attach"),
-                :title     => t,
-                :alt       => t,
-                :enabled   => "vm.saveable(angularForm)",
-                'on-click' => "vm.attachClicked(angularForm)",
-                :primary   => 'true'}
-    %miq-button{:name      => t = _("Reset"),
-                :title     => t,
-                :alt       => t,
-                :enabled   => "!angularForm.$pristine",
-                'on-click' => "vm.resetClicked(angularForm)"}
-    %miq-button{:name      => t = _("Cancel"),
-                :title     => t,
-                :alt       => t,
-                :enabled   => "true",
-                'on-click' => "vm.cancelAttachClicked(angularForm)"}
-
-:javascript
-  ManageIQ.angular.app.value('cloudVolumeFormId', '#{@volume.id}');
-  ManageIQ.angular.app.value('storageManagerId', '#{@volume.ext_management_system.id.to_s}');
-  ManageIQ.angular.app.value('cloudVolumeFormOperation', 'ATTACH');
-  miq_bootstrap('#form_div');
+= render :partial => "layouts/flash_msg"
+.col-md-12
+  - deviceMountpointRequiered = @volume.ext_management_system.type == 'ManageIQ::Providers::Amazon::StorageManager::Ebs' ? true : false
```

Comment:
A. Typo? `Requiered` -> `Required`
B. We shouldn't be doing this type of provider-specific check in the common UI code.  These options should be coming from DDF params in the providers

Reply:
ahhh i see, alrighty ill need to update my api as well then